test/src/test/scala/monocle/function/AtSpec.scala

```diff
@@ -0,0 +1,12 @@
+package monocle.function
+
+import monocle.MonocleSuite
+import monocle.law.discipline.function.AtTests
+
+class AtSpec extends MonocleSuite {
+
+  implicit def mmapAt[K, V]: At[MMap[K, V], K, V] = At.fromIso(MMap.toMap)
+
+  checkAll("fromIso", AtTests.defaultIntIndex[MMap[Int, String], String])
```

Comment:
ideally it should generate indexes and check `LensLaw` for them but I didn't know how to integrate that. So the temporary and ugly solution I found was to use the laws for a fixed index ...